storyscript/Story.py

```diff
@@ -21,6 +22,25 @@ def _parser():
     return Parser()
 
 
+class StoryContext:
+    """
+    Represents context of a given story.
+    """
+
+    def __init__(self, story, features):
+        self.story = story
+        self.features = features
+        self.deprecations = []
+
+    def deprecate(self, tree, name, **kwargs):
+        deprecation = WarningDeprecation.deprecate(
+            self.story, tree=tree, name=name)
+        self.deprecations.append(deprecation)
+
+    def get_deprecations(self):
```

Comment:
Python-style is:

```suggestion
    def deprecations(self):
```

and rename `deprecations` to `_deprecations`